cachi2/core/package_managers/gomod.py

```diff
@@ -771,7 +746,7 @@ def _vet_local_deps(dependencies: List[dict], module_name: str, allowed_patterns
             )
```

Comment:
IMO we can remove the `Module <module> wants to replace <package> with a local dependency: <path>`  log message as well, it's just noise at this point

Reply:
log message removed